test/unit/reporting/test_eval_output_cells.py

```diff
@@ -411,7 +411,7 @@ def test_eval_output_cell_custom_dataset(self):
         eval_output = EvalOutput(
             eval_name="summarization_accuracy",
             dataset_name="Dataset 1",
-            prompt_template="prompt",
+            prompt_template=DEFAULT_PROMPT_TEMPLATE,
```

Comment:
Nit: can you perhaps use a different "custom" template here to make it more clear that the prompt template is a user-provided one?

Reply:
Sure, will update the test case.